packages/nimbus/src/components/tooltip/tooltip.mdx

```diff
@@ -13,36 +11,123 @@ menu:
 tags:
   - component
   - media
+figmaLink: >-
+  https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1695-45519&m
 ---
 
-# Tooltip
+# Tooltips
+
+---
+
+Tooltips display informative text when users hover over or focus on an element.
+
+## Overview
+
+Tooltips offer concise, non-obtrusive guidance and supplemental information, enhancing user understanding and interaction with UI elements.
+
+### Resources
+
+Deep dive on details and access design library.
+
+[TECH RELATED LINK](https://example.com)
+[TECH RELATED LINK](https://example.com)
```

Comment:
I don't think we need these just yet

Reply:
these should be links to the react aria tooltip:
https://react-spectrum.adobe.com/react-spectrum/Tooltip.html#tooltip

and the w3c aria tooltip pattern:
https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/